dotfiles.py

```diff
@@ -1,29 +1,52 @@
 #!/usr/bin/env python3.8
-"""Module docstring."""
+
 import filecmp
+from collections import namedtuple
+from os.path import exists
 from os.path import expanduser
 from shutil import copyfile
 from typing import List
-from typing import Tuple
+from typing import NamedTuple
 
 import click
 
-DOTFILES: List[Tuple[str, str]] = [("~/.tmux.conf", "tmux.conf"), ("~/.vimrc", "vimrc")]
+
+class Dotfile(NamedTuple):
+    system_path: str
```

Comment:
Instead of using a string, you could use an actual [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) object.